openmc/mgxs/mgxs.py

```diff
@@ -50,6 +50,7 @@
 # Filter types corresponding to each domain
 _DOMAIN_TO_FILTER = {'cell': openmc.CellFilter,
                      'distribcell': openmc.DistribcellFilter,
+                     'sum(distribcell)': openmc.DistribcellFilter,
```

Comment:
You can also average/sum over other domains, right?  Take a look at `MGX.num_subdomains`.  I was trying to handle these guys in #725 by stripping the `avg()` form the name before indexing `_DOMAIN_TO_FILTER`.  Not sure what's the better solution, but we should probably stick to one or the other.